src/pages/product-details/comments/comments-item/reply-form/reply-form.js

```diff
@@ -69,7 +68,7 @@ const ReplyForm = ({ userFirstName, user, cancel, commentId, refetchComments, pr
           <span>
             <span>{userFirstName}</span>
             <ReplyOutlinedIcon className={styles.replyIcon} />
-            <span>{firstName}</span>
+            <span>{user?.firstName}</span>
```

Comment:
If we don't have a user we should not reach this code.

Reply:
when the user is deleted, I add a message: "Deleted User", instead of his name
![image](https://user-images.githubusercontent.com/79531224/150513640-8bd0e88c-4546-4ae2-a8de-7aac211d3c7a.png)